### PR TITLE
add search on EAN in product

### PR DESCRIPTION
**Pull request by chafique-delli**
add search on EAN in product

### CODE REVIEW

**Comment by bealdav** [comment]
:+1:

File: product_gtin/product_view.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<openerp>
+    <data>
+
+<record id="product_template_search_view" model="ir.ui.view">
+    <field name="model">product.template</field>
+    <field name="inherit_id" ref="product.product_template_search_view"/>
+    <field name="arch" type="xml">
+        <field name="name" position="replace">
```
Comment (pedrobaeza) [inline product_gtin/product_view.xml]: Instead of replacing, you can use position="attributes" and modify only filter_domain attribute.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by lmignon** [comment]
:+1:

**Pull request merged**